### PR TITLE
Updates model edit to require only input of updated fields

### LINKED ISSUE #13: Edit model should not demand all fields
Python client currently demands data for all the fields when updating an existing model. This should be updated to allow only the changed parameter to be specified.

### PR DESCRIPTION
**Pull request by appukuttan-shailesh**
Fixes #13 
`edit_model()` now requires only input of updated fields (previously all fields had to be specified)

### CODE REVIEW

**Comment by appukuttan-shailesh** [comment]
@apdavison : Ready now

File: hbp_validation_framework/__init__.py [hunk 1]
```diff
@@ -793,29 +788,45 @@ def edit_test(self, name="", test_id="", alias=None, version="", author="",
...
+        if test_id == "":
+            raise Exception("Test ID needs to be provided for editing a test.")
 
         id = test_id   # as needed by API
         test_data = locals()
-        for key in ["self", "test_id", "values"]:
+        for key in ["self", "test_id"]:
             test_data.pop(key)
 
+        # assign existing values for parameters not specified
```
Comment (apdavison) [inline hbp_validation_framework/__init__.py]: This shouldn't really be necessary. The PUT url should accept missing parameters. Or we could define a PATCH endpoint.

Comment (apdavison) [reply hbp_validation_framework/__init__.py]: Please could you open a ticket in the VF repository for modifying the API accordingly?

Comment (appukuttan-shailesh) [reply hbp_validation_framework/__init__.py]: Done.... https://github.com/HumanBrainProject/hbp-validation-framework/issues/181

**Pull request merged**